51c.md

```diff
@@ -0,0 +1,16 @@
+NIP-51b
+======
+
+Hashtag List
+-------------------------
+
+`draft` `optional` `author:fiatjaf` `author:arcbtc` `author:monlovesmango` `author:eskema` 
+
+An event with kind `30002` is defined as a list event (see [NIP-51 - Lists](51.md)) for hashtags. The list of tags should follow the format for "t" tags as suggested in [NIP-12 - Generic Tag Queries](12.md#suggested-use-cases).
+
+Commonly established list parameter names for this kind are:
+- `mute` - a list of tags that the user wants to mute
+- `follow` - a list of tags that the user wants to follow
+- `bookmark` - a list of tags that the user wants to bookmark
```

Comment:
```suggestion
- `mute` - a list of Hashtags that the user wants to mute
- `follow` - a list of Hashtags that the user wants to follow
- `bookmark` - a list of Hashtags that the user wants to bookmark
```

Reply:
again this is a list of tags. tag has been explicitly defined for 51c. 

"The list of tags should follow the format for "t" tags as suggested in [NIP-12 - Generic Tag Queries](https://github.com/monlovesmango/nips/blob/lists/12.md#suggested-use-cases)."